app/models/concerns/hyrax/basic_metadata.rb

```diff
@@ -19,6 +19,8 @@ module BasicMetadata
       # Used for a license
       property :license, predicate: ::RDF::Vocab::DC.rights
 
+      property :rights_notes, predicate: ::RDF::URI.new('http://purl.org/dc/elements/1.1/rights'), multiple: false
```

Comment:
@ruthtillman Would your expectation be that this should be `multiple: true`? In reading your comment (https://github.com/samvera/hyrax/issues/3294), it appears that this may need to be multi-valued.

Reply:
Yes, in some cases it may need to be multi-valued. It's a matter of best practice (probably single) vs. handling edge cases (almost certainly multi-valued).